colorful-dream/beamerthemecolorful-dream.sty

```diff
@@ -362,10 +362,10 @@
             {\sectionbannerfalse\printBibCommand}\null\vfill\null
         \fi
     }%
-    \hspace*{-1em}\parbox{\dimexpr\textwidth+2em}{\parbox[b]{.35\textwidth}{%
+    \hspace*{-1em}\parbox{\dimexpr\textwidth+2em}{\parbox[b]{.5\textwidth}{%
         {\small\textbf{\insertauthor}}\par
         {\footnotesize\btcd@outro{}}%
-    }\hfill\parbox[b]{.35\textwidth}{%
+    }\hfill\parbox[b]{.5\textwidth}{%
     {\null\hfill\footnotesize\href{mailto:\btcd@email}{\btcd@email}}%
```

Comment:
You could also (instead?) change this line to:
`{\null\hfill\footnotesize\href{mailto:\btcd@email}{\hbox{\btcd@email}}}%`